lib/diagnostic.js

```diff
@@ -7,13 +7,23 @@
 // (uppercase strings without punctuation)
 // and returns that array.
 const normalizeWords = function (/* */) {
-  /* your response here */
+
+
+function returnSplitString(string){
+   return string.split(" ");
+
 }
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
 const uniqueWords = function (/* */) {
-  /* your response here */
+
```

Comment:
This looks like it would be a good answer for question 4, but here you actually want to create an array rather than an object.